tweets/async-hooks.tweet

```diff
@@ -0,0 +1,3 @@
+If you use async_hooks with #nodejs please read our recent blog post and comment on the Github issue (linked in blog post) so that we have your use case in the list. Thanks!
```

Comment:
```suggestion
If you use async_hooks with Node.js, please see our recent blog post and comment on the GitHub issue (linked in blog post) so that we can account for your use case as we work on new APIs:
```

updated to not use the hashtag (hashtags are effectively useless on Twitter unless they're for something unique like an event) and be more specific in the way the input will be used. Also no need to say thanks.